workspaces/website/src/components/Layout/PageLayout.tsx

```diff
@@ -18,6 +25,17 @@ type Props = {
 };
 
 export const PageLayout = (props: Props) => {
+  const [isSummitPopupOpen, setIsSummitPopupOpen] = useLocalStorage('isSummitPopupOpen', true);
+  const [isPopupOpen, setIsPopupOpen] = useState(false);
```

Comment:
@milosyuki Isn't isPopupOpen state redundant because we can use  isSummitPopupOpen? Or did using `isSummitPopupOpen` display popup before page is loaded fully?

Reply:
Yes, it was being displayed before the page was fully loaded so that's why I had to introduce additional state.